torch/csrc/jit/python/script_init.cpp

```diff
@@ -1502,6 +1502,14 @@ void initJitScriptBindings(PyObject* module) {
       .def_property_readonly(
           "name",
           [](const StrongFunctionPtr& self) { return self.function_->name(); })
+      .def(
+          "_set_ignore_amp",
```

Comment:
do we need anything like this for modules?

Reply:
no, since the use case here is just aot autograd